spec/niso/jats/article_spec.rb

```diff
@@ -11,22 +11,360 @@
     it_behaves_like "a serializer"
   end
 
-  Dir[File.join(__dir__, "../../fixtures/niso-jats/**/Smallsamples/*.xml")].each do |file|
-    version = file.match(/\/(?<version>\d+\.\d\w?\d?)\/Smallsamples/)[:version]
-
-    # Skip files from old draft version
-    next if version == "0.4"
-
-    # All 1.1d3 files contain a bad xmlns for ali that is missing the trailing slash
-    # xmlns:ali="http://www.niso.org/schemas/ali/1.0"
-    next if version == "1.1d3"
-
-    describe "Smallsamples from NISO JATS #{version}" do
-      filename = Pathname.new(file)
-      context "with #{filename.relative_path_from(File.join(__dir__, "../../fixtures"))}" do
-        let(:fixture) { File.new(file) }
-        it_behaves_like "a serializer"
-      end
+  xcontext "with niso-jats/archiving/0.4/Smallsamples/samplesmall-archive0-oasis.xml" do
```

Comment:
I'm trying to avoid using an array to wrap examples in a spec since it will make spec inspection hard. Instead of putting them in an array, I would suggest

1. Keeping them as they are
2. Deleting them directly

I understand using arrays for multiple fixtures may make the spec code concise/succinct, but it also makes it hard to inspect the spec case by case.

Reply:
We can't "Deleting them directly" because they are from another repository that we don't manage. We can only skip them.

Reply:
No, I mean deleting the `xcontext` not the fixtures.

Reply:
Sure, that is fine then.